scio-extra/src/test/scala/com/spotify/scio/extra/hll/HLLSpec.scala

```diff
@@ -0,0 +1,30 @@
+package com.spotify.scio.extra.hll
+
+import com.spotify.scio.testing.PipelineSpec
+
+trait HLLSpec extends PipelineSpec {
+
+  def checkWithErrorRate(actual: Seq[Long], expected: Seq[Long], errorRate: Double): Unit = {
```

Comment:
hm... do you think this would be valuable to publish as part of a Scio artifact for consumers to use in their own tests? Since it's there aren't any HLL-specific imports, maybe it could be moved into `scio-test/src/main` and refactored into a `ProbabilisticSpec` or something like that?

Reply:
Yes, good idea, even though having only these two functions are not that helpful in most of the probabilistic test, better we start with these and add more helper function as it requires.